extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -14,6 +19,58 @@ export class AmpStoryShoppingAttachment extends AmpStoryPageAttachment {
 
     /** @private {?Element} */
     this.container_ = null;
+
+    /** @private @const {?../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+
+    /** @private {?../../../src/service/localization.LocalizationService} */
+    this.localizationService_ = null;
+  }
+
+  /**
+   * Updates the CTA based on the shopping data.
+   * @param {string} pageIndex
+   * @param {string} pageId
+   * @private
+   */
+  updateCta_(pageIndex, pageId) {
+    const shoppingTag = this.element.ownerDocument
+      .getElementById(pageId)
+      .getElementsByTagName('amp-story-shopping-tag');
+    const numShoppingTags = shoppingTag != null ? shoppingTag.length : 0;
```

Comment:
A null check is not necessary. 
If no elements are found, the HTMLCollection will be empty.
https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByTagName.